src/main/java/uk/gov/companieshouse/api/filter/CustomCorsFilter.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.companieshouse.api.filter;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.springframework.web.cors.CorsUtils;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+public class CustomCorsFilter implements Filter {
+
+    private List<String> externalMethods;
+    private String OPTIONS_METHOD = "OPTIONS";
+    private String ACCESS_CONTROL_ALLOW_ORIGIN = "Access-Control-Allow-Origin";
+    private String ACCESS_CONTROL_ALLOW_HEADERS = "Access-Control-Allow-Headers";
+    private String ACCESS_CONTROL_MAX_AGE = "Access-Control-Max-Age";
+    private String ACCESS_CONTROL_ALLOW_METHODS = "Access-Control-Allow-Methods";
```

Comment:
Could off-the-shelf constants be used for the ACCESS_CONTROL strings?
eg org.springframework.http.HttpHeaders